docs/changes/cosmology/12136.feature.rst

```diff
@@ -0,0 +1,14 @@
+Flat cosmologies are now set by a mixin class, ``FlatCosmologyMixin`` and its
```

Comment:
I guess this brings up a question: do people who wrote their own `Cosmology` subclasses have to adjust? Or would at least all the old behaviour still work with an unadjusted class?

Reply:
This should be a backward compatible change. Anything that inherits from FlatLamdaCDM will get the new features like equivalence. Defining a FlatX Cosmology without using the mixins will not change any behavior, but won’t get the new features. But they didn’t have them previously, so no change there.

Reply:
OK, great!